wiki/osu!tourney/ko.md

```diff
@@ -1,349 +1,15 @@
 # osu!tourney
 
-**osu!tourney** 은 멀티 플레이 경기를 동시에 관전하고, 스트리밍 하기 위해 제작된 multi-spectator 클라이언트입니다.
+**osu!tourney** 는 공식 다중 관전용 클라이언트로, 멀티플레이 방에 있는 모든 플레이어의 플레이를 보여주고 방송 송출하기 위하여 제작된 클라이언트 입니다.
 
-이 프로그램를 엑세스 할 수 있는 권한은 토너먼트 스텝만 가능합니다.
-토너먼트 스텝이 되기 위해서는 **tournaments@ppy.sh** 에 연락하여 승인을 받아야합니다.
-**osu!tourney**의 사용기한은 토너먼트(대회)가 끝나는 시점으로 종료됩니다.
+이 클라이언트를 사용하면서 **문제에** 직면한 경우 [troubleshooting guide](/wiki/osu!tourney/Troubleshooting "Troubleshooting") 를 읽거나  [tournaments@ppy.sh](mailto:tournaments@ppy.sh) 로 메일을 보내주세요.
 
-**토너먼트 프로그램 사용시 스트리밍을 하는 대회, 멀티 이벤트 이외의 클라이언트 사용을 금지합니다. 또한 클라이언트는 개인용으로 사용할 수 없습니다.**
+## 관련 게시글
 
```

Comment:
translate the words in quotes as well

Reply:
asking for the example of quotes please? I don't actuallly know what quote exactly means.
Are you talking about like 'acronym' or 'BufferTime' like this?
waiting for your reply <3

Reply:
Completed! I didn't know what you were meaning :(

Thanks :)